packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -113,7 +113,9 @@ class MediaQueryData {
   int get hashCode => hashValues(size, devicePixelRatio, textScaleFactor, padding);
 
   @override
-  String toString() => '$runtimeType(size: $size, devicePixelRatio: $devicePixelRatio, textScaleFactor: $textScaleFactor, padding: $padding)';
+  String toString() => '$runtimeType(size: $size, '
```

Comment:
nit: avoid => if you have a line break in the expression

Reply:
Done.